data-transfer/src/main/java/it/eng/datatransfer/model/TCKRequest.java

```diff
@@ -0,0 +1,71 @@
+package it.eng.datatransfer.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import it.eng.tools.model.DSpaceConstants;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class TCKRequest implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 4451699406021801277L;
+
+    @JsonProperty(DSpaceConstants.AGREEMENT_ID)
```

Comment:
Do we need JsonProperty here, with same value like field?
Reflection should work here, when converting to java